operator/pkg/util/merge_iop.go

```diff
@@ -50,8 +50,6 @@ type istioOperatorSpec struct {
 type istioComponentSetSpec struct {
 	Base            *baseComponentSpec `json:"base" patchStrategy:"merge"`
 	Pilot           *componentSpec     `json:"pilot" patchStrategy:"merge"`
-	Policy          *componentSpec     `json:"policy" patchStrategy:"merge"`
```

Comment:
Does this break operator API compatibility during upgrade?

Reply:
I think istioctl should complain and fail to run, so that user could notice and remove those mixer related options?

Reply:
Not sure myself.

Reply:
@richardwxn ^^

Reply:
this won't break it, I think it would already fail from previous changes